eogrow/pipelines/rasterize.py

```diff
@@ -252,5 +255,9 @@ def _get_dataset_path(self, filename: str) -> str:
     def _get_output_features(self) -> List[FeatureSpec]:
         """Lists all features that are to be saved upon the pipeline completion"""
         features: List[FeatureSpec] = [FeatureType.BBOX]
+
+        if self.vector_feature[0].is_temporal():
+            features.extend([FeatureType.TIMESTAMP])
```

Comment:
any reason for extend instead of `append`?

Reply:
none whatsoever